core/src/main/java/hudson/logging/LogRecorder.java

```diff
@@ -148,7 +148,7 @@ public AutoCompletionCandidates doAutoCompleteLoggerName(@QueryParameter String
     }
 
     @Restricted(NoExternalUse.class)
-    transient /*almost final*/ RingBufferLogHandler handler = new RingBufferLogHandler() {
+    transient /*almost final*/ RingBufferLogHandler handler = new RingBufferLogHandler(256) {
```

Comment:
You can use the static (now public) field of `RingBufferLogHandler` instead of hard-coded 256